### PR TITLE
Editorial Changes

### PR DESCRIPTION
**Pull request by wareid**
Per @iherman's wonderful recommendations, some changes and amendments to the audiobooks spec.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,9 +55,9 @@ <h1>Introduction</h1>
 
 			<p>This specification is a module of the <a>Web Publication</a> specification. It will describe the requirements to create an Audiobook profile of a Web Publication.</p>
 
-			<p>An Audiobook is defined as a collection of audio resources grouped together by a reading order, metadata, and resources, all contained in a manifest. This Audiobook can live on the Open Web Platform, or packaged using the <a href="https://w3c.github.io/wpub/index.htmlpackaged-web-publications">Packaged Web Publications</a> specification.</p>
+			<p>An Audiobook is defined as a collection of audio resources grouped together by a reading order, metadata, and resources, all contained in a manifest. This Audiobook can live on the Open Web Platform, or packaged using a specification to be determined by the Working Group.</p>
```
Comment (mattgarrish) [inline index.html]: A minor word of caution would be to make "to be determined by the working group" an ednote, as writing it into the spec prose has a way of leading to it being forgotten over time.

File: index.html [hunk 2]
```diff
@@ -55,9 +55,9 @@ <h1>Introduction</h1>
...
 
-			<p>This specification is intended to bring clear standards to the audiobooks distribution model on the web. It should facilitate different user agent architectures for the consumption of Audiobooks. The primary goal is to bring clarity to a part of the publishing industry currently underserved by standards, while opening Audiobooks to the Open Web Platform and new user agents.</p>
+			<p>This specification is intended to bring clear standards to the audiobooks distribution model on the web and between businesses. It should facilitate different user agent architectures for the consumption of Audiobooks. The primary goal is to bring clarity to a part of the publishing industry currently underserved by standards, while opening Audiobooks to the Open Web Platform and new user agents.</p>
```
Comment (mattgarrish) [inline index.html]: "clear standards" is one of those subjective kinds of claims that has a way of arousing passions. Just saying it "standardizes" might be all you need here.

File: index.html [hunk 3]
```diff
@@ -97,29 +97,17 @@ <h3>Bounds</h3>
...
 
-					<p>The table of contents is expressed via an [[html]]&#160;element (typically a <a href="https://www.w3.org/TR/html/sections.html#the-nav-element"><code>nav</code> element</a>) in one of the <a href="https://w3c.github.io/wpub/indexwp-resources">resources</a>. This element MUST be identified by the <code>role</code> attribute&#160;[[html]] value "<code>doc-toc</code>"&#160;[[dpub-aria-1.0]], and MUST be the first element in the document &#8212; in <a href="https://dom.spec.whatwg.org/#concept-tree-order">document tree order</a>&#160;[[dom]] &#8212; with that <code>role</code> value.</p>
-
-					<p>If the table of contents is not located in the <a href="https://w3c.github.io/wpub/index#wp-primary-entry-page">primary entry page</a>, the manifest SHOULD <a href="#table-of-contents">identify the resource</a> that contains the structure.</p>
+					<p>The Web Publication specification [[wpub]] defines the <a href="https://w3c.github.io/wpub/index.html#wp-table-contents">general structure</a> of the Table of Contents, defined in HTML [[html]].</p>
```
Comment (mattgarrish) [inline index.html]: Minor nit, but "define" is used twice in the sentence. I think you can just drop the comma and second instance so it reads  "defines the general structure ... in HTML"

**Comment by mattgarrish** [comment]
Although, it looks like there might be a bigger problem with pr-preview right now: https://github.com/tobie/pr-preview/issues/30

**Comment by wareid** [comment]
@mattgarrish I addressed your comments (and tried to tackle the links, but I will run it through the link checker again once merged).

**Pull request merged**